### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by aarabii**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,143 @@
...
+    return a + b;
+}
+```
+
+`Main function`
+```aura
+fn main() {
+    let result = add(5, 7);
+    println("The result is {}", result);
+}
```
Comment (nahharris) [inline README.md]: Change this to:

```
main {
    result = add(5, 7);
    println("The result is ${result}");
}

File: README.md [hunk 2]
```diff
@@ -1,13 +1,143 @@
...
+
+## Example Code
+
+Here is a simple example of Aura code from the [Aura repository](https://github.com/auralangco/aura/blob/master/examples/core.aura):
+
+`Function to add two numbers`
+```aura
+fn add(a: int, b: int): int {
+    return a + b;
+}
```
Comment (nahharris) [inline README.md]: Change this to 

```aura
fn add(a Int, b Int) -> Int = a + b;
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,143 @@
...
+   cargo build
+   ```
+
+## Usage
+
+To compile an Aura program, use the following command:
+
+```sh
+cargo run -- path/to/yourfile.aura
+```
```
Comment (nahharris) [inline README.md]: At the moment the front-end isn't implemented, so compiling aura source code isn't yet supported the user writes the IR directly. Don't think it's worth mentioning it at the moment

**Review by nahharris — CHANGES_REQUESTED** [review]

**Comment by aarabii** [comment]
all the unused and unnecessary and wrong examples were changed

**Comment by nahharris** [comment]
This closes #2

**Pull request merged**